R/ParameterList.R

```diff
@@ -132,6 +132,7 @@ as.list.ParameterList <- function(x, ...) {
 #' Getter functions for the slots of a [`ParameterList`] object
 #' @inheritParams ParameterList-Shared
 #' @family ParameterList
+#' @param nchains (`integer`) \cr the number of chains.
```

Comment:
or `n_chains`? just a question, best to go with what fits in naming pattern used so far